.projenrc.js

```diff
@@ -109,27 +109,23 @@ docgenTask.exec('jsii-docgen -l typescript -o docs/typescript.md');
 docgenTask.exec('jsii-docgen -l python -o docs/python.md');
 docgenTask.exec('jsii-docgen -l java -o docs/java.md');
 
-const hooks = project.addTask('hooks');
-hooks.exec('./git-hooks/setup.sh');
+for (const spec of [LATEST_SUPPORTED_K8S_VERSION, LATEST_SUPPORTED_K8S_VERSION - 1, LATEST_SUPPORTED_K8S_VERSION - 2].map(s => new Number(s))) {
+  const backportTask = project.addTask(`backport:${spec}`);
+  backportTask.exec(`npx backport --accesstoken \${GITHUB_TOKEN} --pr \${BACKPORT_PR_NUMBER} --branch k8s-${spec}/main --noFork --prTitle "{commitMessages}"`);
+}
 
 // backport PR's to other branches
+// this doesn't use the backport tasks because there's a dedicated
+// action for it which already invokes it.
 // see https://github.com/tibdex/backport
-const backport = project.github.addWorkflow('backport');
-backport.on({ pullRequest: { types: ['closed'] } });
-backport.addJob('backport', {
+const backportWorkflow = project.github.addWorkflow('backport');
+backportWorkflow.on({ pullRequest: { types: ['closed'] } });
+backportWorkflow.addJob('backport', {
   runsOn: 'ubuntu-18.04',
   permissions: {
     contents: JobPermission.WRITE,
   },
   steps: [
-    {
```

Comment:
We can remove this now that https://github.com/sqren/backport/pull/336 is merged and released

Reply:
open source w00t!